OpenKh.Patcher/PatcherProcessor.cs

```diff
@@ -279,5 +284,124 @@ private static void PatchAreaDataScript(Context context, List<AssetFile> sources
 
             Kh2.Ard.AreaDataScript.Write(stream.SetPosition(0), scripts.Values);
         }
+
+        private static readonly Dictionary<string, byte> characterMap = new Dictionary<string, byte>(){
+            { "Sora", 1 }, { "Donald", 2 }, { "Goofy", 3 },  { "Mickey", 4 },  { "Auron", 5 }, { "PingMulan",6 }, { "Aladdin", 7 },  { "Sparrow", 8 }, { "Beast", 9 },  { "Jack", 10 },  { "Simba", 11 }, { "Tron", 12 }, { "Riku", 13 }, { "Roxas", 14} 
+        };
+
+        private static readonly IDeserializer deserializer = new DeserializerBuilder().IgnoreUnmatchedProperties().Build();
+            
+        private static void PatchListReplace(Context context, List<AssetFile> sources, Stream stream)
+        {
+            foreach (var source in sources)
+            {
+                    string sourceText = File.ReadAllText(context.GetSourceModAssetPath(source.Name));
+                    switch (source.Type)
+                    {
+                        case "trsr":
+                            var trsrList = Kh2.SystemData.Trsr.Read(stream).ToDictionary(x => x.Id, x => x);
+                            var moddedTrsr = deserializer.Deserialize<Dictionary<int, Kh2.SystemData.Trsr>>(sourceText);
+                            foreach (var treasure in moddedTrsr)
+                            {
+                                trsrList[treasure.Value.Id].ItemId = treasure.Value.ItemId;
+                            }
+                                Kh2.SystemData.Trsr.Write(stream.SetPosition(0), trsrList.Values);
+                            break;
+
+                        case "item":
+                            var itemList = Kh2.SystemData.Item.Read(stream);
+                            var moddedItem = deserializer.Deserialize<Kh2.SystemData.Item>(sourceText);
+                            if (moddedItem.Items1 != null)
```

Comment:
This was my bad... `Items1` should be `Items` and `Items2` should be `Stats`. Do you want to rename them? Do a CTRL+R+R to ensure you are renaming it everywhere in the codebase.

Reply:
Addressed in https://github.com/Xeeynamo/OpenKh/pull/457/commits/136d8d16700419aa10cc79c44aa29c15f5fb2335